src/view/com/composer/GifAltText.tsx

```diff
@@ -114,61 +114,83 @@ export function GifAltText({
 }
 
 function AltTextInner({
-  onSubmit,
+  altTextRef,
+  control,
   link,
   params,
-  initialValue: initalValue,
 }: {
-  onSubmit: (text: string) => void
+  altTextRef: React.MutableRefObject<string>
+  control: DialogControlProps
   link: AppBskyEmbedExternal.ViewExternal
   params: EmbedPlayerParams
-  initialValue: string
 }) {
-  const {_} = useLingui()
-  const [altText, setAltText] = useState(initalValue)
-  const control = Dialog.useDialogContext()
-
-  const onPressSubmit = useCallback(() => {
-    onSubmit(altText)
-  }, [onSubmit, altText])
+  const t = useTheme()
+  const {_, i18n} = useLingui()
+  const [altText, setAltText] = useState(altTextRef.current)
```

Comment:
we shouldn’t read refs during render. 

this might be ok but why are we using a ref here? isn’t this a controlled input anyway?